support-frontend/assets/pages/digital-subscription-checkout/components/endSummary/endSummarySelector.js

```diff
@@ -10,6 +10,7 @@ import { getBillingDescription } from 'helpers/productPrice/priceDescriptionsDig
 export type EndSummaryProps = {
   priceDescription: string,
   promotion: Option<?string>,
+  endSummaryMobile?: boolean,
```

Comment:
I couldn't see this new prop `endSummaryMobile` in use, is it required?

Reply:
No - I added it for the previous solution I had to this problem. I'll remove it. 👍